uint32[] active_index_
float64 active_rate_

uint32[] number_of_neurons_
float64[] activity_
